packages/remix-server-runtime/server.ts

```diff
@@ -259,7 +259,7 @@ async function handleDocumentRequestRR(
     );
   } catch (error: unknown) {
     if (serverMode !== ServerMode.Test) {
-      console.log("Error in entry.server handleDocumentRequest:", error);
+      console.error("Error in entry.server handleDocumentRequest:", error);
```

Comment:
`console.error` is what we used to use - and explains why we started seeing these in integration tests since we only blew away `console.error`